### ISSUE TITLE
Default Tag For both sonarr and radarr

### ISSUE THREAD

**Issue by hackmonker** [petio-team/petio#674]
Right now we can apply filters and add tags according to the filters. However if none of the movies and shows match the filter they get sent to the default root folder and profile set in the arr settings which is fine but unfortunately since I work a lot with tags these remain untagged as there is no default tag present unlike root paths and profile. So a default tag option might solve this problem.

Solution Suggestion 

Maybe an extra settings of Tag here can solve this problem. Not sure if this will require a lot of work. hopefully it should not but thanks anyway.

![image](https://user-images.githubusercontent.com/46002790/149507235-3b8d6e5e-3c06-4b20-bc1f-c063f9083bc8.png)

**Comment by JacobBrown4** [comment]
A work around I found is you can just set a generic filter that applies to any possible media and only be attaching a tag with it. Mine is if the year is greater than 1900. I'd really like if I could tag stuff by user still though.
<img width="778" alt="Petio" src="https://user-images.githubusercontent.com/68622976/150693164-8bf48480-b959-4961-b6e0-84ae3001f39c.png">

**Comment by perrottg** [comment]
> A work around I found is you can just set a generic filter that applies to any possible media and only be attaching a tag with it. Mine is if the year is greater than 1900. I'd really like if I could tag stuff by user still though. <img alt="Petio" width="778" src="https://user-images.githubusercontent.com/68622976/150693164-8bf48480-b959-4961-b6e0-84ae3001f39c.png">



> Right now we can apply filters and add tags according to the filters. However if none of the movies and shows match the filter they get sent to the default root folder and profile set in the arr settings which is fine but unfortunately since I work a lot with tags these remain untagged as there is no default tag present unlike root paths and profile. So a default tag option might solve this problem.
> 
> Solution Suggestion
> 
> Maybe an extra settings of Tag here can solve this problem. Not sure if this will require a lot of work. hopefully it should not but thanks anyway.
> 
> ![image](https://user-images.githubusercontent.com/46002790/149507235-3b8d6e5e-3c06-4b20-bc1f-c063f9083bc8.png)

I need this as well as I would like all tv series added to sonarr with the tag "265" by default.

**Comment by hackmonker** [comment]
> A work around I found is you can just set a generic filter that applies to any possible media and only be attaching a tag with it. Mine is if the year is greater than 1900. I'd really like if I could tag stuff by user still though. <img alt="Petio" width="778" src="https://user-images.githubusercontent.com/68622976/150693164-8bf48480-b959-4961-b6e0-84ae3001f39c.png">

Yeah that can work but that tags all movies if I am not wrong. so if maybe animation movie is tagged as animated, this will also tag that movie. My case is tagging the ones that does not match any filters explicitly.
